chingu-coders/Voyage2-Turtles-02
# updated styles

## Purpose
* Update CSS with matching styles

## Approach/Fixes
* Decrease border radius to 3px on all features
* Cleaned up `footer` css
* Added labels for color picker and toolbox to match app

## Testing
* Pull into your local device
* See that all CSS code is matching and features look consistent
